react-frontend/src/components/RegisterForm/RegisterForm.js

```diff
@@ -27,7 +27,7 @@ const RegisterForm = () => {
       await register(email, password, name, surname)
       navigate('/login')
     } catch (err) {
-      setError(err.message)
+      setError(err.error || 'An unexpected error occurred')
```

Comment:
Do you mean `err.message`?

Reply:
Yes, Refactored in [b655d78](https://github.com/YaroslavKSE/AuctionWeb/pull/3/commits/b655d78aa4cf807c011c5b5bb5f81bf9be9e0e84)